crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -389,6 +392,33 @@ impl<T: WasmModule> ModuleHostActor for WasmModuleHostActor<T> {
         self.instances.seed().scheduler.close();
         self.instances.join()
     }
+
+    fn one_off_query(
+        &self,
+        caller_identity: Identity,
+        respond_to: oneshot::Sender<Result<Vec<spacetimedb_lib::relation::MemTable>, DBError>>,
+        query: String,
+    ) {
+        let db = &self.instances.shared.seed.worker_database_instance.relational_db;
+        let auth = AuthCtx::new(self.instances.shared.seed.info.identity, caller_identity);
+        // TODO(jgilles): make this a read-only TX when those get added
+        let mut tx = db.begin_tx();
```

Comment:
Should this maybe use the automatic-rollback-RAII thing, especially in case there's more control flow for error handling here later?

Reply:
When I didn't `db.end_tx()`, it was leaving the transaction open. I was wondering about adding `#[must_use]` to begin_tx

Reply:
I wasn't aware that existed

Reply:
Do you know where I should look for that?